### PR TITLE
feat(events): add calendar component for events page

### LINKED ISSUE #324: feat(events): create calendar component for events page
image

### PR DESCRIPTION
**Pull request by kanhaiya88628**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #324 

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
image

@kanhaiya88628 make the following changes
-> Increase the gap between weekdays and the days box. check figma for exact details.

image

-> add a events page which must redirect to coming soon page.

File: client/src/pages/events.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Head from 'next/head';
+
+// Components
+import Marginals from '../components/marginals/Marginals';
+import EventCalendar from '../components/calendar/EventCalendar';
+
+const AboutPage = () => {
```
Comment (120EE0692) [inline client/src/pages/events.jsx:8]: @kanhaiya88628 change it to events page.

File: client/src/pages/events.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  return (
+    <>
+      <Head>
+        {/* <!-- =============== Primary Meta Tags =============== --> */}
+        <title>About | Monday Morning</title>
```
Comment (120EE0692) [inline client/src/pages/events.jsx:13]: @kanhaiya88628 change about to events.

File: client/src/pages/events.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <meta
+          name='keywords'
+          content='about, monday morning, mondaymorning, monday morning, mm, nit rkl, nit, nit rourkela, nitr, nitrkl, rkl, rourkela'
+        />
+
+        {/* <!-- =============== Open Graph / Facebook =============== --> */}
+        <meta property='og:type' content='website' />
+        <meta
+          property='og:url'
+          content='https://mondaymorning.nitrkl.ac.in/about'
```
Comment (120EE0692) [inline client/src/pages/events.jsx:28]: @kanhaiya88628 change here as well

File: client/src/pages/events.jsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+export async function getServerSideProps() {
+  return {
+    redirect: {
+      destination: '/comingSoon',
+      permanent: false,
+    },
+  };
+}
+
+export default AboutPage;
```
Comment (120EE0692) [inline client/src/pages/events.jsx:86]: @kanhaiya88628 change here ass well

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

**Review by 120EE0692 — APPROVED** [review]

**Comment by rutajdash** [comment]
@120EE0692 since when do we have an events page?

**Review by rutajdash — COMMENTED** [review]
1. Why was an events page created?
2. Why was an add events component created? That should be added to admin panel, not main app and certainly not open to public.
3. What happened to the home page calendar component?

**Comment by 120EE0692** [comment]
> 1. Why was an events page created?
> 2. Why was an add events component created? That should be added to admin panel, not main app and certainly not open to public.
> 3. What happened to the home page calendar component?

@rutajdash 
1. Events page was created to provide information to the general public about all the ongoing events.

We will show calendar as well as event card.
image

2. add events component will not be accessible to general public, we will add permission checks.

3. Home page calendar component UI is ready and already merged.

**Comment by rutajdash** [comment]
> > 1. Why was an events page created?
> > 2. Why was an add events component created? That should be added to admin panel, not main app and certainly not open to public.
> > 3. What happened to the home page calendar component?
> 
> @rutajdash
> 
> 1. Events page was created to provide information to the general public about all the ongoing events.
> 
> We will show calendar as well as event card. image
> 
> 2. add events component will not be accessible to general public, we will add permission checks.
> 3. Home page calendar component UI is ready and already merged.

And why do we need an events page?
Considering that the events section is not a focus point of Monday Morning. Simple popups from the calendar on the landing page with minimal event information should suffice in my opinion.
Furthermore, Monday Morning, or rather Project Infinity is partnered with other projects at NITR, few of which focus on the college events aspect.

**Comment by 120EE0692** [comment]
@kanhaiya88628 changes done?

**Review by 120EE0692 — APPROVED** [review]

**Review by rutajdash — COMMENTED** [review]
Why was chart.js removed from the package dependencies?

**Review by rutajdash — APPROVED** [review]

**Pull request merged**